src/QuickMarcEditor/QuickMarcEditorRows/utils.js

```diff
@@ -67,10 +67,14 @@ const DELETE_EXCEPTION_ROWS = {
 
 const is1XXField = (tag) => tag[0] === '1';
 
-export const hasDeleteException = (recordRow, marcType = MARC_TYPES.BIB) => {
+export const hasDeleteException = (recordRow, marcType = MARC_TYPES.BIB, linksCount = 0) => {
   const rows = DELETE_EXCEPTION_ROWS[marcType];
 
-  if (marcType === MARC_TYPES.AUTHORITY && is1XXField(recordRow.tag)) return true;
+  if (marcType === MARC_TYPES.AUTHORITY) {
+    if (is1XXField(recordRow.tag) || (recordRow.tag === '010' && linksCount > 0)) {
```

Comment:
Checking for `linksCount` will not always be enough
MARC bibs can be controlled by either `001` or `010$a` - we can know which case it is by looking at `authority.naturalId`.
If `authority.naturalId === 010$a` then we have to disable it.
The authority record can be found in `<QuickMarcEditor>` component as `instance` prop

Btw, for MARC Authority records `instance` is the authority records you get when searching in MARC Authority app
For MARC Bib - it's Instance records in Inventory app
And for MARC Holdings - it's a Holdings record